### PR TITLE
Chore:show the current pwd to make the scala-stew works

### PR DESCRIPTION
**Pull request by He-Pin**
Motivation:
Testing the current working directory

### CODE REVIEW

**Comment by mdedetrich** [comment]
@He-Pin Can you also add the relevant xmx https://github.com/apache/pekko-http/blob/3c567c0cc8c10b1eb115d524c5cf6cf5e5f5b62a/.github/workflows/publish-1.0-docs.yml#L49 , this may solve the issue in a nicer way

**Comment by He-Pin** [comment]
@mdedetrich Just updated

File: .github/workflows/scala-steward.yml [hunk 1]
```diff
@@ -11,8 +11,11 @@ jobs:
     name: Launch Scala Steward
     if: github.repository == 'apache/pekko'
     steps:
-      - name: Enable jvm-opts
-        run: cp .jvmopts-ci .jvmopts
+      - name: Setup JVM options
+        run: |-
+          echo $PWD
+        env:
```
Comment (mdedetrich) [inline .github/workflows/scala-steward.yml:17]: This needs to be added to the actual scala-steward action, i.e. line 20

**Comment by mdedetrich** [comment]
> @mdedetrich Just updated

You added it to wrong place, see comment

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**